console/console-server/Dockerfile

```diff
@@ -3,7 +3,7 @@
 # License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
 #
 
-FROM centos:7
+FROM scratch
```

Comment:
Why scratch?

Reply:
It would, I can leave the console server at centos:8 if you wish, the main motivation was to avoid any external dependency and reduce image size and need for respins etc.

Reply:
Note that in the case of the console, the oauth proxy container would still have curl available and access to the same localhost ports.

Reply:
An alternative would be to use some basic alpine image that has curl etc, but with a much smaller footprint than centos...